sql/gnomex_db_upgrade_to_5.36.0.0.sql

```diff
@@ -0,0 +1,5 @@
+use gnomex;
+
+alter table Sample add column libPrepPerformedByID int(10) null;
```

Comment:
With the latest commit I added the foreign key and refactored the name of the new field.  Thanks for bringing this up.